src/symbolDocumenter.ts

```diff
@@ -11,7 +11,7 @@ const labelDefinitionRegex = /^\@?((\$\$(?!\.))?[\w\.]+)(?::|\s|$)/;
 const parentLabelRegex = /^(((\@|\$\$)(?!\.))?\w[\w\.]*)(?::|\s|$)/;
 const evalExpressionRegex = /^\@?([\w\.]+)\:?\s+(=|equ|eval)\s+(.+)(;.*)?$/i;
 const defineExpressionRegex = /^\@?([\w\.]+)\:?\s+(inc(?:bin|hob|trd)|b?include|insert|binary|def[bdlmsw]|d[bcdszw]|abyte[cz]?|byte|word|dword)\s+([^$;]+)(;.*)?$/i;
-const keywordRegex = /^(equ|eval|org|end?t|align|phase|dephase|unphase|shift|save(?:bin|hob|sna|tap|trd)|empty(?:tap|trd)|inc(?:bin|hob|trd)|b?include|insert|binary|end|output|tap(?:out|end)|fpos|page|slot|size|cpu|device|encoding|charset|proc|macro|local|shared|public|rept|dup|block|endm|endp|edup|exitm|module|endmod(?:ule)?|(?:un)?define|export|disp|textarea|map|field|defarray|assert|fatal|error|warning|message|display|shellexec|def[bdlmsw]|d[bcdszw]|abyte[cz]?|byte|word|dword|if|ifdef|ifndef|ifused|ifnused|else|elseif|endif|adc|add|and|bit|call|ccf|cp|cpdr?|cpir?|cpl|daa|dec|[de]i|djnz|exx?|halt|im|in|inc|indr?|inir?|j[pr]|ld|lddr?|ldir?|neg|nop|or|ot[di]r|out|out[di]|pop|push|res|ret[in]?|rl|rla|rlc|rlca|rld|rr|rra|rrc|rrca|rrd|rst|sbc|scf|set|sli?a|sll|swap|sra|srl|sub|xor|amsdos|breakpoint|buildcpr|buildsna|run|save|setcpc|setcrtc|print|fail|repeat|rend|until|switch|case|default|break|endswitch|stop|while|wend|incl48|incl49|inclz4|inczx7|incexo|lz48|lz49|lz4|lzw7|lzexo|lzclose|align|bank|bankset|limit|protect|write direct|charset|def[irs]|str|struct)$/i;
+const keywordRegex = /^(equ|eval|org|end?t|align|phase|dephase|unphase|shift|save(?:bin|hob|sna|tap|trd)|empty(?:tap|trd)|inc(?:bin|hob|trd)|b?include|insert|binary|end|output|tap(?:out|end)|fpos|page|slot|size|cpu|device|encoding|charset|proc|macro|local|shared|public|rept|dup|block|endm|endp|edup|exitm|module|endmod(?:ule)?|(?:un)?define|export|disp|textarea|map|field|defarray|assert|fatal|error|warning|message|display|shellexec|def[bdlmsw]|d[bcdszw]|abyte[cz]?|byte|word|dword|if|ifdef|ifndef|ifused|ifnused|else|elseif|endif|adc|add|and|bit|call|ccf|cp|cpdr?|cpir?|cpl|daa|dec|[de]i|djnz|exx?|halt|im|in|inc|indr?|inir?|j[pr]|ld|lddr?|ldir?|neg|nop|or|ot[di]r|out|out[di]|pop|push|res|ret[in]?|rl|rla|rlc|rlca|rld|rr|rra|rrc|rrca|rrd|rst|sbc|scf|set|sli?a|sll|swap|sra|srl|sub|xor|amsdos|breakpoint|buildcpr|buildsna|run|save|setcpc|setcrtc|print|fail|repeat|rend|until|switch|case|default|break|endswitch|stop|while|wend|incl48|incl49|inclz4|inczx7|incexo|lz48|lz49|lz4|lzw7|lzexo|lzclose|align|bank|bankset|limit|protect|write direct|charset|def[irs]|str|struct|endstruct)$/i;
```

Comment:
- `align` and `charset` double defined
- `def[irs]` - merge character enumerations with already defined -> **`def[bdlmswir]`**
- whitespace between `write` and `direct` should be matched nicely and "less-whitespace-strict" with `\\s+`